laser_encoders/download_models.py

```diff
@@ -88,7 +88,14 @@ def download_laser2(self):
         self.download("laser2.cvocab")
 
     def download_laser3(self, lang: str, spm: bool = False):
-        lang = self.get_language_code(LASER3_LANGUAGE, lang)
+        result = self.get_language_code(LASER3_LANGUAGE, lang)
+
+        if isinstance(result, list):
+            raise ValueError(
+                f"The 'get_language_code' function returned a list for lang='{lang}'. Please choose a specific script from the list: {result}"
```

Comment:
I don't think we need to expose the function name to the user. We can simply say something like:

```
f"There are script-specific models available for {lang}. Please choose one from the following: {result}"
```